packages/renderer/src/lib/preferences/PreferencesResourcesRendering.spec.ts

```diff
@@ -135,6 +135,6 @@ test('Expect to be start, delete actions disabled and stop, restart enabled when
 test('Expect to see the no resource message when there is no providers', async () => {
   providerInfos.set([]);
   render(PreferencesResourcesRendering, {});
-  const panel = screen.getByLabelText('no-resource-panel');
+  const panel = screen.getByText('No resources found');
```

Comment:
There was an extra parent div on the EmptyScreen just to contain this aria-label, and it was causing issues with vertical centering. Instead of adding a class="h-full", I saw we weren't using this pattern elsewhere and removed the div, then had to fix the test to look for the text instead.

It was a judgement call; I could add back the div with class, or update EmptyScreen to accept an aria-label.

Reply:
I think aria label would be helpful because getByText can be anything (could be displayed at a different place than what we expect)

Reply:
Reverted test, added support for aria-label to EmptyScreen and added the label back there to avoid requiring the extra div.